clusterloader2/testing/watch-list/config.yaml

```diff
@@ -0,0 +1,84 @@
+name: huge-secrets
+tuningSets:
+- name: Uniform10qps
+  qpsLoad:
+    qps: 10
+steps:
+- name: Create well known namespaces
```

Comment:
I decided to manually create ns because the app (which will be defined in a separate PR) needs to know the target ns

Reply:
Alternatively you may want to use "namespaces" with specified "prefix: huge-services": https://github.com/kubernetes/perf-tests/blob/4da7097e35b13225ed05bf3dfcb842864c11fae4/clusterloader2/api/types.go#L140-L140

Then you namespace should be named "huge-services-0".

This will avoid using "namespaceRange: null" which is slightly hacky :)

Reply:
oh, didn't know i can to that, that would be awesome, let me try:)

Reply:
looks like it works, thanks for the tip!